Content.Client/Administration/UI/Logs/AdminLogsControl.xaml.cs

```diff
@@ -103,6 +111,16 @@ private void PlayerSearchChanged(LineEditEventArgs args)
 
     private void LogSearchChanged(LineEditEventArgs args)
     {
+        // This exception is thrown if the regex is invalid, which happens often, so we ignore it.
+        try
+        {
+            LogSearchRegex = new Regex("(" + LogSearch.Text + ")", RegexOptions.IgnoreCase);
```

Comment:
I feel like this should have a timeout (though probably make it a private static readonly up top):
```suggestion
            LogSearchRegex = new Regex("(" + LogSearch.Text + ")", RegexOptions.IgnoreCase, TimeSpan.FromSeconds(1));
```
I'm also still concerned about this confusing admins who (understandably) don't know what regex is and are trying to search for someone saying '1+1' and the search not finding that.